lib/prawn/table.rb

```diff
@@ -479,8 +479,8 @@ def initial_row_on_initial_page
 
       # If there isn't enough room left on the page to fit the first data row
       # (including the header), start the table on the next page.
-      @pdf.bounds.move_past_bottom
-
+      #@pdf.bounds.move_past_bottom
```

Comment:
Just remove the line. Leaving dead code as comments pollutes the file, and one can always look in the commit history to see the "blame"